<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="welcome_page">
       <property name="styleSheet">
        <string notr="true">QWidget {
	background: #fff;
}

QPushButton {
	color: black;
	background: white;
	border-radius: 5px;
	padding: 10px;
	border: 2px solid #333;
}

QPushButton:pressed {
	background: #888;
	color: #fff;
}</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="wecome_layout">
          <item row="0" column="1">
           <widget class="QPushButton" name="input_data_button">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Input Data</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="upload_data_button">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Upload Data</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="input_data_page">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid #888;
	border-radius: 5px;
	color: #000;
	background: #fff;
	padding: 5px 10px;
}

QWidget {
	background: #fff;
}</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="input_data_layout">
          <item row="2" column="3">
           <widget class="QPushButton" name="done_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Done</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QTableWidget" name="input_table">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>833</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto Mono</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #000;
background: #fff;</string>
            </property>
            <row>
             <property name="text">
              <string>1</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Email</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Phone</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Number of Courses</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Course 1</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Course 2</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Course 3</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Course 4</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Monday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Tuesday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Wednesday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Thursday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Friday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Saturday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>K1</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>K2</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>K3</string>
             </property>
            </column>
            <item row="0" column="1">
             <property name="text">
              <string/>
             </property>
             <property name="font">
              <font>
               <family>Roboto Mono</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="3">
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="back_button_2">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="upload_data_page">
       <property name="styleSheet">
        <string notr="true">QWidget {
	background: #fff;
}

QLabel {
	color: #555;
}

QPushButton {
	color: #000;
	background: #fff;
	padding: 5px 10px;
	border-radius: 5px;
	border: 1px solid #333;
}

QPushButton:pressed {
	background: #888;
	color: #fff;
}</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="upload_data_layout">
          <item row="3" column="4">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="4" colspan="3">
           <widget class="QLineEdit" name="teacher_data_path">
            <property name="font">
             <font>
              <family>Roboto Mono</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #000;
background: #ddd;
padding: 5px 10px;
border: 1px solid #888;
border-radius: 5px;</string>
            </property>
            <property name="text">
             <string>Z:/some path/some more</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <widget class="QLabel" name="upload_student_label">
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Upload Student data (csv file)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="7">
           <widget class="QPushButton" name="student_data_browse_button">
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #000;
background: #fff;
padding: 5px 10px;
border-radius: 5px;
border: 1px solid #333;</string>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
          <item row="7" column="5">
           <widget class="QPushButton" name="back_button">
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="upload_teacher_label">
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Upload Teacher data (csv file)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="7">
           <widget class="QPushButton" name="teacher_data_browse_button">
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #000;
background: #fff;
padding: 5px 10px;
border-radius: 5px;
border: 1px solid #333;</string>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
          <item row="7" column="7">
           <widget class="QPushButton" name="generate_tt_button">
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Generate TT</string>
            </property>
           </widget>
          </item>
          <item row="6" column="7">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="4">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="4" colspan="3">
           <widget class="QLineEdit" name="student_data_path">
            <property name="font">
             <font>
              <family>Roboto Mono</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #000;
background: #ddd;
padding: 5px 10px;
border: 1px solid #888;
border-radius: 5px;</string>
            </property>
            <property name="text">
             <string>Z:/some path/something else</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="search_tt_page">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	color: #000;
	background: #fff;
	padding: 5px 10px;
	border-radius: 5px;
	border: 1px solid #333;
}

QPushButton:pressed {
	background: #888;
	color: #fff;
}

QWidget {
	background: #fff;
}</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QGridLayout" name="search_tt_layout">
          <item row="3" column="3">
           <widget class="QLineEdit" name="teacher_name_input">
            <property name="font">
             <font>
              <family>Roboto Mono</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 5px 10px;
color: #000;
background: #fff;
border: 1px solid #888;
border-radius: 5px;</string>
            </property>
            <property name="text">
             <string>TeacherName</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="role_label">
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 5px 10px;
color: #000;
background: #fff;</string>
            </property>
            <property name="text">
             <string>Role</string>
            </property>
           </widget>
          </item>
          <item row="5" column="7">
           <widget class="QPushButton" name="search_tt_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Search TT</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="6">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="select_role_dropdown">
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 2px solid #333;
border-radius: 5px;
color: #000;
background: #fff;
padding: 5px 10px;</string>
            </property>
            <item>
             <property name="text">
              <string>Teacher</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Student</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="teacher_input_label">
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 5px 10px;
color: #000;
background: #fff;</string>
            </property>
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="8">
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>80</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="5">
           <widget class="QPushButton" name="back_to_upload_page_button">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Product Sans</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
